src/main/kotlin/com/github/mitchellolsthoorn/testgenie/coverage/TestGenieCoverageRenderer.kt

```diff
@@ -0,0 +1,84 @@
+package com.github.mitchellolsthoorn.testgenie.coverage
+
+import com.intellij.codeInsight.hint.HintManager
+import com.intellij.codeInsight.hint.HintManagerImpl
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.editor.LogicalPosition
+import com.intellij.openapi.editor.ex.EditorGutterComponentEx
+import com.intellij.openapi.editor.markup.ActiveGutterRenderer
+import com.intellij.openapi.editor.markup.LineMarkerRendererEx
+import com.intellij.ui.HintHint
+import com.intellij.ui.LightweightHint
+import com.intellij.ui.components.JBLabel
+import com.intellij.util.ui.FormBuilder
+import java.awt.Color
+import java.awt.Graphics
+import java.awt.Rectangle
+import java.awt.event.MouseEvent
+
+class TestGenieCoverageRenderer(private val color: Color, private val lineNumber : Int, private val tests : List<String>) : ActiveGutterRenderer,
```

Comment:
KDoc on the class?

Reply:
This should be addressed in our most recent commit.